src/components/trade/TradesTable/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { Order } from 'api/operator'
+import styled, { css } from 'styled-components'
+import { RowWithCopyButton } from 'components/orders/RowWithCopyButton'
+import React from 'react'
+import { SimpleTable } from '../../common/SimpleTable'
+import TradesTableContext from './Context/TradesTableContext'
+import { DateDisplay } from 'components/orders/DateDisplay'
+import { faExchangeAlt, faQuestion } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { OrderSurplusDisplay } from 'components/orders/OrderSurplusDisplay'
+import { constructPrice } from 'utils'
+import { BlockExplorerLink } from 'apps/explorer/components/common/BlockExplorerLink'
+import { COLOURS } from 'styles'
+import { Theme } from 'theme'
+import { variants } from 'styled-theming'
+import TokenImg from 'components/common/TokenImg'
+import { TokenErc20 } from '@gnosis.pm/dex-js'
+
+const { white, blackLight } = COLOURS
+
+const Icon = styled(FontAwesomeIcon)`
+  background: ${({ theme }): string => theme.grey}33; /* 33==20% transparency in hex */
+  border-radius: 1rem;
+  width: 2rem !important; /* FontAwesome sets it to 1em with higher specificity */
+  height: 2rem;
+  padding: 0.4rem;
+  margin-left: 0.5rem;
+  cursor: pointer;
+`
+
+const IconOutline = styled(FontAwesomeIcon)`
+  background: none;
+  border-radius: 1rem;
+  border: 2px solid;
+  width: 2rem !important; /* FontAwesome sets it to 1em with higher specificity */
+  height: 2rem;
+  padding: 0.4rem;
+  margin-left: 0.5rem;
+  cursor: pointer;
+`
+
+export const TableTheme = variants('mode', 'variant', {
+  default: {
+    [Theme.LIGHT]: css`
+      color: ${blackLight} !important;
+    `,
+    [Theme.DARK]: css`
+      color: ${white} !important;
+    `,
+  },
+  get primary() {
+    return this.default
+  },
+})
+
+const TableRow = styled.tr`
+  padding: 6px 2px 6px 2px !important;
+  th {
+    font-weight: 600 !important;
+  }
+`
+const StyledTableRow = styled(TableRow)`
+  /* Fold in theme css above */
+  ${TableTheme}
+`
+
+export const TradesTableHeader = (): JSX.Element => {
+  const tableContext = React.useContext(TradesTableContext)
+
+  return (
+    <StyledTableRow variant={'default'}>
+      <th>
+        Tx Id <IconOutline icon={faQuestion} />
```

Comment:
We can reuse this icon?
https://protocol-explorer.dev.gnosisdev.com/rinkeby/orders/0x3bdceef7965b0f342aa37f7489007cd4ce2c9a07b485571351f133943d23d4cab6bad41ae76a11d10f7b0e664c5007b908bc77c961144d83
![image](https://user-images.githubusercontent.com/622217/129250660-9eb45bb7-5428-485e-bc81-7ad8eb3f3f17.png)